scripts/monitoring/cron-send-s3-metrics.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+'''
+  Collect and send AWS S3 bucket stats
+'''
+# vim: expandtab:tabstop=4:shiftwidth=4
```

Comment:
I believe the vim stuff has to be within the first 5 lines, so having it exactly at line 5 is a bit scary. What if, for instance, you decide to make the module doc string longer and that pushes this down a few lines?

Please consider moving this line above the doc string comment.